src/main/java/org/broadinstitute/dsm/route/BSPKitQueryRoute.java

```diff
@@ -46,11 +45,11 @@ public Object handle(Request request, Response response) throws Exception {
         }
 
         return TransactionWrapper.inTransaction(conn -> {
-            return getBSPKitInfo(conn, kitLabel, response);
+            return getBSPKitInfo(conn, kitLabel, response, this.notificationUtil);
```

Comment:
having DB connection directly in route is not a good idea, new `DAO` class for it would be good.